### ISSUE TITLE
SonoffBridge Device display problem on Start Page

### ISSUE THREAD

**Issue by ascillato** [reloxx13/TasmoAdmin#97]
Hi,

On the START page, the device that is **SonoffBridge** shows the moving icon as if SonWEB could not connect to it, while on the LIST page it is connected.

That device doesn't have any output set.

![image](https://user-images.githubusercontent.com/35405447/40074620-7647dade-5850-11e8-8dd2-e3e9e397db50.png)

![image](https://user-images.githubusercontent.com/35405447/40075171-04df98f8-5852-11e8-90c2-fc237150c8bd.png)

```
15:00:23 CMD: status 0
15:00:23 MQT: stat/bridge/STATUS = {"Status":{"Module":25,"FriendlyName":["SonoffBridge"],"Topic":"bridge","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"SaveData":1,"SaveState":1,"ButtonRetain":0,"PowerRetain":0}}
15:00:23 MQT: stat/bridge/STATUS1 = {"StatusPRM":{"Baudrate":19200,"GroupTopic":"sonoffs","OtaUrl":"http://sonoff.maddox.co.uk/tasmota/sonoff.ino.bin","RestartReason":"Software/System restart","Uptime":"0T00:32:08","StartupUTC":"2018-05-15T17:28:15","Sleep":0,"BootCount":23,"SaveCount":37,"SaveAddress":"F7000"}}
15:00:23 MQT: stat/bridge/STATUS2 = {"StatusFWR":{"Version":"5.12.0m","BuildDateTime":"2018-04-22T02:58:55","Boot":6,"Core":"2_4_1","SDK":"2.2.1(cfd48f3)"}}
15:00:23 MQT: stat/bridge/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":2,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["NetWireless","NetWireless"],"TelePeriod":300,"SetOption":["00008009","55818000"]}}
15:00:23 MQT: stat/bridge/STATUS4 = {"StatusMEM":{"ProgramSize":509,"Free":492,"Heap":15,"ProgramFlashSize":1024,"FlashSize":1024,"FlashMode":3}}
15:00:23 MQT: stat/bridge/STATUS5 = {"StatusNET":{"Hostname":"bridge-5530","IPAddress":"192.168.1.3","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.1.1","Mac":"DC:4F:22:CB:35:9A","Webserver":2,"WifiConfig":2}}
15:00:23 MQT: stat/bridge/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.1.117","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_CB359A","MqttUser":"homeassistant","MqttType":2,"MAX_PACKET_SIZE":1000,"KEEPALIVE":15}}
15:00:23 MQT: stat/bridge/STATUS7 = {"StatusTIM":{"UTC":"Tue May 15 18:00:23 2018","Local":"Tue May 15 15:00:23 2018","StartDST":"Sun Mar 25 02:00:00 2018","EndDST":"Sun Oct 28 03:00:00 2018","Timezone":-3}}
15:00:23 MQT: stat/bridge/STATUS10 = {"StatusSNS":{"Time":"2018-05-15T15:00:23"}}
15:00:23 MQT: stat/bridge/STATUS11 = {"StatusSTS":{"Time":"2018-05-15T15:00:23","Uptime":"0T00:32:08","Vcc":3.499,"Wifi":{"AP":2,"SSId":"NetWireless","RSSI":40,"APMac":"18:D6:C7:80:38:2C"}}}
```

**Comment by Flori1989** [comment]
I can confirm that it is still the case.

**Comment by reloxx13** [comment]
fixed in v1.6.1-beta1
